tasks/Mock Jury.md

```diff
@@ -0,0 +1,42 @@
+# Summary
+The experiment performed was designed to test the effects of group deliberation on jurors when exposed to inadmissable evidence.
+Speciffically the study sought to see how group deliberation would effect jurors individual ability to disregard inadmissable evidence.
+
+# References
+London, K., & Nunez, N. (2000). The effect of jury deliberations on jurors' propensity to disregard inadmissible evidence. Journal of Applied Psychology, 85(6), 932–939. https://doi.org/10.1037/0021-9010.85.6.932
+
+Link: https://psycnet-apa-org.proxy.library.upenn.edu/record/2000-16508-010
+
+# Stimuli
+## The visual components
+N/A 
+
+## Materials for alternative versions of the experiment 
+3 versions of a mock trial script.  All versions of the script should contain testimones from both the defense and the prosecution.  The only difference in the versions of the scipts are the extra admissable/inadmissable evidence. The three versions should be as follows:
+- Control group script, containing a script from a mock trial
+- Admissable group script, the same script as the control group, containing an extra piece of evidence that the group is allowed to use in their decision making
+- Inadmissable group script, the same script as the control group, containing the same extra piece of evidence as the Admissable group, however, this group is told the extra evidence is inadmissable
+
+# Procedure
+## Steps
+All participants will act as jurors.
+1) Jurors are split into groups ranging in size of 8-12. (Average group size should be around 10)
```

Comment:
I would add a sub-bullet here: 
After being divided into groups, each group was exposed to one of three versions of a mock trial script.  All versions of the script should contain testimonies from both the defense and the prosecution.  The only difference in the versions of the scripts is the extra admissible/inadmissible evidence. The three versions should be as follows:
- Control group script, containing a script from a mock trial
- Admissable group script, the same script as the control group, containing an extra piece of evidence that the group is allowed to use in their decision making
- Inadmissible group script, the same script as the control group, containing the same extra piece of evidence as to the Admissable group, however, this group is told the extra evidence is inadmissible